recipes-app/iot2050-firmware-update/iot2050-firmware-update_0.5.bb

```diff
@@ -11,15 +12,23 @@
 DESCRIPTION = "OSPI Firmware Update Scripts"
 MAINTAINER = "chao.zeng@siemens.com"
 
-SRC_URI = "file://iot2050-firmware-update.tmpl"
+SRC_URI = " \
+    file://update.conf.json.tmpl \
+    file://iot2050-firmware-update.tmpl"
 
-TEMPLATE_FILES = "iot2050-firmware-update.tmpl"
+TEMPLATE_FILES = "update.conf.json.tmpl iot2050-firmware-update.tmpl"
 
 inherit dpkg-raw
 
+DEBIAN_DEPENDS = "python3-progress"
+
 do_install() {
+    install -v -d ${D}/usr/lib/iot2050/fwu
+    install -v -m 644 ${WORKDIR}/update.conf.json ${D}/usr/lib/iot2050/fwu/
+    install -v -m 755 ${WORKDIR}/iot2050-firmware-update ${D}/usr/lib/iot2050/fwu/
+
     install -v -d ${D}/usr/sbin/
-    install -v -m 755 ${WORKDIR}/iot2050-firmware-update ${D}/usr/sbin/
+    ln -sf ../lib/iot2050/fwu/iot2050-firmware-update ${D}/usr/sbin/iot2050-firmware-update
```

Comment:
Why that installation into `/usr/lib/iot2050/fwu/` so that we need to link it here?

Reply:
We also have `update.conf.json.tmpl` installed together, which is not very good to install into `sbin`. So the same rules are applied with `eio manager` and some others, which is installed to `/usr/lib`.

Reply:
Sure, conf files should not go into sbin. Maybe they are better off in `/usr/share/iot2050/fwu`?

Reply:
ACK, `/usr/share` is more reasonable.